MC/Config.C

```diff
@@ -46,8 +46,8 @@ static Bool_t  purifyKine      = kTRUE;     // purifyKine flag
 static Bool_t  isFluka         = kFALSE;    // fluka flag
 
 #if ROOT_VERSION_CODE >= ROOT_VERSION(6,0,0)
-#include "MC/DetectorConfig.C"
-#include "MC/GeneratorConfig.C"
+#include "DetectorConfig.C"
```

Comment:
As per [our instructions](https://alice-doc.github.io/alice-analysis-tutorial/analysis/ROOT5-to-6.html#how-can-i-call-macros-inside-macros) one should do:

```c++
R__ADD_INCLUDE_PATH("MC")
#include "DetectorConfig.C"
#include "GeneratorConfig.C"
```

I honestly don't know whether this works (maybe it only works with cling, don't know), can you check?

Reply:
Hi Dario, that would not change anything. The
#include "DetectorConfig.C"
already works, the R__ADD_INCLUDE_PATH is not needed if you do not use the "MC/..." as include. The problem is that ROOT 5 uses gROOT->LoadMacro("$ALIDPG_ROOT/MC/...."), so it uses the absolute path.
And I just tested R__ADD_INCLUDE_PATH("$ALIDPG_ROOT..."), but it doesn't resolve the variable. #include "$ALIDPG_ROOT/MC/..." doesn't work either. I guess the preprocessor doesn't resolve environment variables at all, so we cannot do it on the preprocessor level.

Again, the current PR is working fine for me. I am just not sure whether it has any side effects, but I guess not.

What I meant originally is to set
ROOT_INCLUDE_PATH+=$ALIDPG_ROOT
in dpgsim.sh, such that ROOT searches in the $ALIDPG_ROOT, and then use
#include "MC/DetectorConfig.C".
However, I just tried that and it doesn't work either. I have no idea why. Is the include path not respected for .C files?

So my suggestion is to go with the PR as is, and I would add a comment to the includes that this was changed and one might need the full path and explain the problem. (So in case the include fails and someone checks the code, he knows immediately.)

Reply:
Can you just specify whether the current PR works _both for ROOT5 and ROOT6_?

Reply:
The includes do not change at all for ROOT5, they are inside the #if ROOT_VERSION_CODE >= ROOT_VERSION(6,0,0) fence.
Well, the fixing of variables and typos affect ROOT5, but they have been wrong before anyway. But I didn't explicitly test with ROOT5.